client/src/partials/journal/modals/journalPosting.modal.js

```diff
@@ -0,0 +1,124 @@
+angular.module('bhima.controllers')
+  .controller('JournalPosterModalController', JournalPosterModalController);
+
+JournalPosterModalController.$inject = [ 
+  '$uibModalInstance', 'SessionService', 'JournalPostingModalService',
+  'GridGroupingService', 'Records', 'GridColumnService', 'NotifyService',
+  '$location'
+];
+
+/**
+ * @module journal/modals/JournalPoster.modal
+ *
+ * @description
+ * This controller provides a tool to do trial balance
+ */
+function JournalPosterModalController(ModalInstance, Session, journalPostingModalService, Grouping,  records, Columns, Notify, $location) {
+  var vm = this;
+  var columns = [
```

Comment:
You don't have either `ui-grid-edit` or `ui-grid-cellNav` as directives.  You don't need `enableCellEdit` and `allowCellFocus` properties on these, I don't think.